spec/forms/decidim/participatory_documents/suggestion_form_spec.rb

```diff
@@ -34,17 +46,17 @@ module ParticipatoryDocuments
       end
 
       it "is invalid with a body that is too short" do
-        form.body = "Short body"
+        form.body = "body"
 
         expect(form).to be_invalid
-        expect(form.errors[:base]).to include(I18n.t("activemodel.errors.models.suggestion.attributes.too_short", min_length: SuggestionForm::MIN_BODY_LENGTH))
+        expect(form.errors[:body]).to include(I18n.t("activemodel.errors.models.suggestion.attributes.too_short", min_length: Decidim::ParticipatoryDocuments.config.min_suggestion_length))
```

Comment:
```suggestion
        expect(form.errors[:body]).to include(I18n.t("activemodel.errors.models.suggestion.attributes.too_short", min_length: 5))
```

It is better to check for the final value, so we are sure that this is the default value